detail.py

```diff
@@ -0,0 +1,31 @@
+from django_mako_plus import view_function, jscontext
+from django.http import HttpResponseRedirect
+from catalog import models as cmod
+import math
```

Comment:
You could remove unnecessary imports

Reply:
Good point! I just copied another file and left it there.

Reply:
I agree with Christian here.